### PR TITLE
docs: bump number of built-ins

### PR DESCRIPTION
**Pull request by anderseknert**
:)

Signed-off-by: Anders Eknert <anders@eknert.com>

### CODE REVIEW

File: docs/content/kubernetes-primer.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ Often you don't want to invent new variable names for iteration.  OPA provides t
 
 ### Builtins
 
-On line 5 the *builtin* `startswith` checks if one string is a prefix of the other.  The builtin `sprintf` on line 6 formats a string with arguments.  OPA has 50+ builtins detailed at [openpolicyagent.org/docs/policy-reference](../policy-reference).
+On line 5 the *builtin* `startswith` checks if one string is a prefix of the other.  The builtin `sprintf` on line 6 formats a string with arguments.  OPA has 150+ builtins detailed at [openpolicyagent.org/docs/policy-reference](../policy-reference).
```
Comment (srenatus) [inline docs/content/kubernetes-primer.md:190]: That link is wonky. Let's do
```suggestion
On line 5 the *builtin* `startswith` checks if one string is a prefix of the other. 
The builtin `sprintf` on line 6 formats a string with arguments.
OPA has 150+ builtins detailed at [the Policy Reference](../policy-reference/).
```

**Review by srenatus — APPROVED** [review]
LGTM 📈

File: docs/content/kubernetes-primer.md (hunk 1 above)
Comment (srenatus) [reply docs/content/kubernetes-primer.md:190]: we could also link straight to `../policy-reference/#built-in-functions`

Comment (anderseknert) [reply docs/content/kubernetes-primer.md]: Fixed!

**Review by srenatus — APPROVED** [review]

**Pull request merged**